cli/integration_tests/no_args.t

```diff
@@ -19,21 +19,22 @@ Make sure exit code is 2 when no args are passed
     unlink      Unlink the current directory from your Vercel organization and disable Remote Caching
   
   Options:
-        --version                   
-        --skip-infer                Skip any attempts to infer which version of Turbo the project is configured to use
-        --api <API>                 Override the endpoint for API calls
-        --color                     Force color usage in the terminal
-        --cpuprofile <CPU_PROFILE>  Specify a file to save a cpu profile
-        --cwd <CWD>                 The directory in which to run turbo
-        --heap <HEAP>               Specify a file to save a pprof heap profile
-        --login <LOGIN>             Override the login endpoint
-        --no-color                  Suppress color usage in the terminal
-        --preflight                 When enabled, turbo will precede HTTP requests with an OPTIONS request for authorization
-        --team <TEAM>               Set the team slug for API calls
-        --token <TOKEN>             Set the auth token for API calls
-        --trace <TRACE>             Specify a file to save a pprof trace
-        --verbosity <COUNT>         Verbosity level
-    -h, --help                      Print help
+        --version                         
+        --skip-infer                      Skip any attempts to infer which version of Turbo the project is configured to use
+        --api <API>                       Override the endpoint for API calls
+        --color                           Force color usage in the terminal
+        --cpuprofile <CPU_PROFILE>        Specify a file to save a cpu profile
+        --cwd <CWD>                       The directory in which to run turbo
+        --heap <HEAP>                     Specify a file to save a pprof heap profile
+        --login <LOGIN>                   Override the login endpoint
+        --no-color                        Suppress color usage in the terminal
+        --preflight                       When enabled, turbo will precede HTTP requests with an OPTIONS request for authorization
+        --remote-cache-timeout <TIMEOUT>  Set a timeout for all HTTP requests
+        --team <TEAM>                     Set the team slug for API calls
+        --token <TOKEN>                   Set the auth token for API calls
+        --trace <TRACE>                   Specify a file to save a pprof trace
+        --verbosity <COUNT>               Verbosity level
+    -h, --help                            Print help
```

Comment:
What is the cause of all these diffs? They look visually identical and it doesn't look like a tabs / spaces issue? CLRF?